velox/type/DecimalUtilOp.h

```diff
@@ -239,10 +237,7 @@ class DecimalUtilOp {
       if constexpr (std::is_same_v<R, int64_t>) {
         auto result = convertToInt64(result_large, overflow);
         auto remainder = convertToInt64(remainder_large, overflow);
-        if (!(result >= DecimalUtil::kLongDecimalMin &&
-              result <= DecimalUtil::kLongDecimalMax)) {
-          *overflow = true;
-        } else {
+        if (!overflow) {
```

Comment:
int64_t should always bigger than the minimum of int128_t and less then the maximum value of int128_t... and it seems the `overflow` has been updated inside `convertToInt64`.